plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -150,12 +167,13 @@ private void updateSelectedVersion() {
       try {
         Path home = ManagedCloudSdk.newManagedSdk().getSdkHome();
         version = getSdkVersion(home);
+        isManaged = true;
       } catch (UnsupportedOsException ex) {
         // shouldn't happen but if it does we'll just leave
         // version set to Unknown
       }
     }
-    
+    updateSdk.setEnabled(isManaged);
```

Comment:
This may fit better in `updateControlEnablement()` conceptually?

Reply:
Good observation 👍